operate/cli.py

```diff
@@ -886,19 +844,6 @@ async def _update_service(request: Request) -> JSONResponse:
 
         return JSONResponse(content=output.json)
 
-    @app.put("/api/v2/services")
-    @with_retries
-    async def _update_all_services(request: Request) -> JSONResponse:
-        """Update all services of matching the public id referenced in the hash."""
-        if operate.password is None:
-            return USER_NOT_LOGGED_IN_ERROR
-
-        manager = operate.service_manager()
-        template = await request.json()
-        updated_services = manager.update_all_matching(service_template=template)
-
-        return JSONResponse(content=updated_services)
-
```

Comment:
The idea of this endpoint was to easily update when Pearl supports several instances of the same agent.